app/addons/compaction/views.js

```diff
@@ -19,73 +19,10 @@ define([
 ],
 function (app, FauxtonAPI, Compaction) {
 
-  Compaction.Layout = FauxtonAPI.View.extend({
-    template: 'addons/compaction/templates/layout',
+  // This is no longer used. We need to come up with a solution to allow to compact a specific view with CouchDB 2.0
+  // Once that is supported we can convert this to React.
```

Comment:
Compaction just works on 15986 - but it works.

Why do we comment all this old code?